integration-tests/steps.go

```diff
@@ -233,4 +233,46 @@ var happyPathSteps = []Step{
 			},
 		},
 	},
+	{
+		action: UnbondTokensAction{
+			chain:      0,
+			unbondFrom: 0,
+			sender:     0,
+			amount:     11111111,
```

Comment:
nit: Why unbonding `11111111` and not `11000000`?

Reply:
Originally this was pattern matched from the delegate tokens action above, but ```10000000``` does indeed work with the arithmetic. I Changed each to your number 👍